### ISSUE TITLE
Download last X days

### ISSUE THREAD

**Issue by browntyler** [icloud-photos-downloader/icloud_photos_downloader#682]
## Summary
Ability to download photos/videos within the last X days. 

## Context
I periodically use this utility to download a backup of my iCloud library locally. I then rename all photos and then back them up to an offsite repository. It would be very convenient to only download images and videos uploaded to my iCloud in the last X days. This would allow me to automate the process every X days and not have to download my entire library each time. 

I am unable to use the sync feature as I rename the files and don’t necessarily store them locally.

**Comment by didamarkovic** [comment]
I would find such a feature useful too.

**Comment by michaela-schw** [comment]
This option would enable my use case as well!

I’d like to sync the library within the first X days, e.g. pictures that were deleted shortly after taking them will be deleted from local storage as well.
For older photos/videos though, I want to be free to delete them from iCloud to save space, but they shall be kept in local storage.

This could be achieved by ignoring every change to files older than X days (specified by new parameter) on both iCloud and local storage.

**Comment by michaela-schw** [comment]
Thanks for the hint, @AndreyNikiforov 

However, I don’t want to remove all older files from iCloud, just the videos which take up a lot of space.

**Comment by AndreyNikiforov** [comment]
> However, I don’t want to remove all older files from iCloud, just the videos which take up a lot of space.

I misinterpreted your statement then:

> For older photos/videos though, I want to be free to delete them from iCloud to save space, but they shall be kept in local storage.

**Comment by michaela-schw** [comment]
> I misinterpreted your statement then:

Right, that wasn’t clear enough. I’d like to be free to manually delete selected older stuff from iCloud without losing the local backup.
